runironic-exporter.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/bash
+
+. /bin/configure-ironic.sh
+
+# TODO(dtantsur): merge these two scripts when/if we no longer support
+# all-in-one container
```

Comment:
Its not quite clear to me what this means, is this referring to when we use a different image for ironic-prometheus-exporter?

Reply:
At this point I'd prefer to avoid adding any new images - we already have two critical tasks (machine-api-opearator integration of the baremetal operator, and cluster hosted ironic) blocked on getting forks of the existing images into openshift, adding any more right at this moment is likely to further delay things, which we really, really do need to avoid :)

That said we can start planning for new images to be used in future, but just not right now when those critical tasks are still pending.